Xamarin.Essentials/Browser/Browser.shared.cs

```diff
@@ -5,10 +5,10 @@ namespace Xamarin.Essentials
 {
     public static partial class Browser
     {
-        public static Task OpenAsync(string uri) =>
-            OpenAsync(uri, BrowserLaunchMode.SystemPreferred);
+        public static Task OpenAsync(string uri, BrowserLaunchOptions options = default) =>
```

Comment:
We must not use default parameters, rather create an overload:

```csharp
Task OpenAsync(string uri) =>
    OpenAsync(uri, null);
Task OpenAsync(string uri, BrowserLaunchMode launchMode) =>
    OpenAsync(uri, new BrowserLaunchOptions(launchMode));
Task OpenAsync(string uri, BrowserLaunchOptions options) =>
    PlatformOpenAsync(uri, options);
```

I don't think there will be an overload that takes both `BrowserLaunchMode` AND `BrowserLaunchOptions` - what is the case where you will set colors and then also say ignore those colors?

Reply:
There is none, I will change this in the next commit.